tests/staticfiles_tests/test_management.py

```diff
@@ -176,6 +176,58 @@ def test_handle_path_notimplemented(self):
         self.assertFileNotFound('cleared.txt')
 
 
+class TestInteractiveMessages(CollectionTestCase):
+    """
+    Test static files collection interactive guidance.
+    """
+
+    def test_no_warning_when_staticdir_does_not_exist(self):
+        stdout = six.StringIO()
+        shutil.rmtree(six.text_type(settings.STATIC_ROOT))
+        call_command('collectstatic', interactive=True, stdout=stdout)
+        self.assertNotIn('overwrite', stdout.getvalue().lower())
+        self.assertNotIn('delete', stdout.getvalue().lower())
+
+    def test_no_warning_for_empty_staticdir(self):
+        stdout = six.StringIO()
+        # Use an empty directory instead of the directory from setUp that contains files
```

Comment:
Note: That's probably note the most elegant thing to do, but I don't think putting it in another TestCase would be more convenient. Ideas are welcome.